main/src/shortcuts/Shortcuts.ts

```diff
@@ -223,15 +223,27 @@ export class Shortcuts {
 }
 
 function pressKeysToCopyItemText (pressedModKeys: string[] = [], showModsKey: string) {
-  let keys = mergeTwoHotkeys('Ctrl + C', showModsKey).split(' + ')
+  const isMac = process.platform === 'darwin';
+  const copyModifier = isMac ? 'Meta' : 'Ctrl';
+  let keys = mergeTwoHotkeys(`${copyModifier} + C`, showModsKey).split(' + ')
   keys = keys.filter(key => key !== 'C' && !pressedModKeys.includes(key))
 
   for (const key of keys) {
     uIOhook.keyToggle(UiohookKey[key as UiohookKeyT], 'down')
   }
 
+  const allowedModifierKeys: Set<UiohookKeyT> = new Set(["Ctrl", "Alt", "Meta", "Shift"]);
+  const modifierKeys: number[] = keys
+      .filter((key) => allowedModifierKeys.has(key as UiohookKeyT))
+      .map((key) => UiohookKey[key as UiohookKeyT])
+
   // finally press `C` to copy text
-  uIOhook.keyTap(UiohookKey.C)
+  uIOhook.keyTap(
+    UiohookKey.C,
+    // On Mac, robotjs requires the modifiers to be specified in this way to
+    // register. See https://github.com/octalmage/robotjs/issues/208#issuecomment-223828356
+    isMac ? modifierKeys : undefined
```

Comment:
I don't use robotjs anymore, are u sure this is required? Becuase it will mean you can't chain price checks like
Ctrl + D, D, D, D, D

Reply:
Actually, I tested a bit more. It looks like we don't actually need to add the modifier keys on the `keyTap` call. I did discover that on Mac, the `pressedModKeys` still need to be toggled to copy Advanced Mod Descriptions for some reason. I don't totally understand why, but it works!